CHANGELOG.md

```diff
@@ -1,204 +1,8 @@
-<a name="3.1.0"></a>
```

Comment:
Can we keep the changelog? It provides historical info after we forked from the chrome launcher.

Reply:
No problem. I've restored the changelog now.